main/serve.js

```diff
@@ -32,28 +50,35 @@ foam.CLASS({
   ],
 });
 
-let server = pkg.Server.create({
-  port: 8080,
-});
-
 const logger = foam.log.ConsoleLogger.create();
-let credentials = null;
-try {
-  credentials = require('../.local/credentials.json');
-} catch (e) {
-  logger.warn(`No Cloud Datastore credentails found. Using read-only copy of
-                   local data snapshot`);
+const modeString = process.argv[2];
```

Comment:
Position-based command lines are fairly brittle. I don't know if there's a good (or any) flag parsing libraries, and I wouldn't recommend writing one for this, so there may not be anything to be done here.

Either way, it can be in a later change.